grades/migrations/0002_test_data.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from django.contrib.auth.models import User
+        from grades.models import Grade, Course
+
+        username = "ido"
+        password = "123456"
+        user1 = User.objects.create_user(username=username, password=password)
+        course1 = Course.objects.create(course_name="course1", credinitials="3")
+        course2 = Course.objects.create(course_name="course2", credinitials="2")
+        test_data = [
+            (user1, course1, 85, 'winter', 2021),
+            (user1, course2, 60, 'spring', 2020),
+            (user1, course1, 20, 'summer', 2090),
+        ]
```

Comment:
Did you verify this works? I would think that you would have to pass in A, B or C instead of winter, spring, summer